sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/main/resources/META-INF/services/org.apache.shardingsphere.orchestration.center.api.ConfigCenter

```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 #
 
-org.apache.shardingsphere.orchestration.center.instance.NacosInstance
+org.apache.shardingsphere.orchestration.center.instance.NacosConfigInstance
```

Comment:
Why rename nacos as "NacosConfigInstance"?  What is the name for zookeeper, as it is used for config center, register center and distributed lock management?

Reply:
nacos can as config center & register center, doesn't support distributed lock.

nacos is different for config center & register center, so we should have two class.

`NacosConfigInstance` is for Config center, `NacosRegistryInstance` will for register center.